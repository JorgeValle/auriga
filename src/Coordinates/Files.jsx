import React from 'react';

// Components should represent one piece of data model
class Files extends React.Component {

  // It only has a render method, because this is a static version of app
  // To make UI interactive, we need to be able to trigger changes to underlying data model
  render() {
    return (
      <div className="files">
        <div className="file">
          <span className="file-letter">a</span>
        </div>
        <div className="file">
          <span className="file-letter">b</span>
        </div>
        <div className="file">
          <span className="file-letter">c</span>
        </div>
        <div className="file">
          <span className="file-letter">d</span>
        </div>
        <div className="file">
          <span className="file-letter">e</span>
        </div>
        <div className="file">
          <span className="file-letter">f</span>
        </div>
        <div className="file">
          <span className="file-letter">g</span>
        </div>
        <div className="file">
          <span className="file-letter">h</span>
        </div>
      </div>
    );
  }
}

export default Files;