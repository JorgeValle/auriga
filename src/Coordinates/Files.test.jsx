import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import Files from './Files.jsx';

// We shallow render to make sure changes in children don't cause cascading fails

configure({
  adapter: new Adapter()
});

describe('<Files />', () => {

  it('shallow renders', () => {
    const wrapper = shallow(<Files />);
  });

});

