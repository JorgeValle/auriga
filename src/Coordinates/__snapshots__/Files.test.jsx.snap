// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Files /> matches snapshot 1`] = `ShallowWrapper {}`;
